otwrapy/_otwrapy.py

```diff
@@ -574,7 +609,7 @@ def __init__(self, wrapper, backend='multiprocessing', n_cpus=-1, verbosity=10,
 
         # This configures how to run samples on the model :
         if self.n_cpus == 1:
-            self._exec_sample = self.wrapper
+            self._exec_sample = _exec_sample_unit_progress_bar(self.wrapper, verbosity)
 
```

Comment:
lets actually allow a "progress_bar" backend keyword, so the condition rewrites `if backend=="progress_bar" or self.n_cpus == 1`
it should be renamed "serial" though which is more explicit (rename the function _exec_sample_serial too)